modules/text_server_adv/text_server_adv.cpp

```diff
@@ -6126,16 +6126,14 @@ Glyph TextServerAdvanced::_shape_single_glyph(ShapedTextDataAdvanced *p_sd, char
 }
 
 _FORCE_INLINE_ void TextServerAdvanced::_add_featuers(const Dictionary &p_source, Vector<hb_feature_t> &r_ftrs) {
```

Comment:
While we're here, we could fix the typo :sweat_smile:

Reply:
Damn, I was hoping to get away without a full recompile :D